submarine-server/server-api/src/main/java/org/apache/submarine/server/api/spec/Environment.java

```diff
@@ -20,18 +20,43 @@
 package org.apache.submarine.server.api.spec;
 
 public class Environment {
-  private String image;
-
+  
+  /**
+   * ID of the environment
+   */
+  private int environmentId;
+  
   /**
-   * Get the image for the experiment. If the {@link ExperimentTaskSpec#getImage()} not
-   * specified the image replaced with it.
-   * @return image
+   * Name of the environment
    */
```

Comment:
It looks like the name is still here.

Reply:
Taken care. Class itself is redundant, so removed.